docs/CODE_OF_CONDUCT.md

```diff
@@ -1,3 +0,0 @@
-303 See Other
-
-Location: https://www.elastic.co/community/codeofconduct
```

Comment:
This allows github.com to link directly to our Elastic code of conduct from its UI. We copied this from how other Elastic repos are set up. Can you add this file back?